wagtail/admin/forms/pages.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, *args, **kwargs):
         can_publish = kwargs.pop('can_publish')
         super().__init__(*args, **kwargs)
         self.fields['new_title'] = forms.CharField(initial=self.page.title, label=_("New title"))
-        self.fields['new_slug'] = forms.SlugField(initial=self.page.slug, label=_("New slug"))
+        self.fields['new_slug'] = forms.SlugField(initial=self.page.slug, label=_("New slug"), allow_unicode=True)
```

Comment:
I think it would be better to make this dependent on the `WAGTAIL_ALLOW_UNICODE_SLUGS` setting:

https://docs.wagtail.io/en/stable/reference/settings.html#unicode-page-slugs

Reply:
Sure!